### PR TITLE
Set KMP_INIT_AT_FORK to FALSE in likwid module to avoid

### PR DESCRIPTION
**Pull request by pramodk**
issue with likwid + Intel compiler.

(see https://github.com/RRZE-HPC/likwid/issues/157 and https://bbpteam.epfl.ch/project/issues/browse/HELP-8691)

### CODE REVIEW

**Review by tristan0x — APPROVED** [review]
LGTM

File: sysconfig/bbpv/tools/modules.yaml [hunk 1]
```diff
@@ -76,6 +76,10 @@ modules:
       - '%clang'
       - '%pgi'
       - 'cmake'
+    likwid:
+      environment:
+        set:
+          KMP_INIT_AT_FORK: 'FALSE'
```
Comment (ohm314) [inline sysconfig/bbpv/tools/modules.yaml:82]: Just out of curiosity.. 
https://github.com/RRZE-HPC/likwid/releases
If you look at the changelog of 4.3.2 to me it seems that this env variable is set by likwid itself... it doesn't hurt to set it in the module too.

**Review by ohm314 — APPROVED** [review]
LGTM

File: sysconfig/bbpv/tools/modules.yaml (hunk 1 above)
Comment (pramodk) [reply sysconfig/bbpv/tools/modules.yaml:82]: thanks for pointing that out. I put constraint ` likwid@:4.3.1` to restrict it up to 4.3.1.

**Pull request merged**